spring-kafka/src/test/java/org/springframework/kafka/kstream/KafkaStreamsJsonSerializationTests.java

```diff
@@ -0,0 +1,210 @@
+/*
+ * Copyright 2017-2018 the original author or authors.
```

Comment:
And this one is new, therefore should be only with the current year.

Reply:
In regards to copyright years this project looks to have a different policy with Spring Boot. I’m just curious which one is recommended legally.

See https://github.com/spring-projects/spring-boot/blob/master/spring-boot-project/spring-boot-actuator-autoconfigure/src/main/java/org/springframework/boot/actuate/autoconfigure/metrics/MeterRegistryPostProcessor.java

Reply:
I can't speak for boot's policy, but we were advised by legal that the copyright should initially just have the year the file was created.

Reply:
@garyrussell Thanks for the quick response!

That's weird as Spring Boot looks to use the year the project has been created as the starting year and I thought the members in the two projects are employed by the same company, hence having the same legal team.

/cc @philwebb